src/unity/python/turicreate/toolkits/_pre_trained_models.py

```diff
@@ -238,3 +238,19 @@ def get_model_path(self, format):
             )[0]
 
         return model_path
+
+class DrawingClassifierPreTrainedModel(object):
+    def __init__(self, model_url = None):
+        filename = "drawing_model_v1.params"
```

Comment:
"drawing model" is extremely vague. Let's give this some more thought before we actually post anything. Anything we post for 5.4 will probably be closer to "v0" than "v1", too.

Reply:
Tracked in #1489